vector/src/main/res/layout/item_bottom_sheet_title.xml

```diff
@@ -16,6 +16,7 @@
         android:layout_height="wrap_content"
         android:textColor="?riotx_text_primary"
         android:textSize="18sp"
+        android:textStyle="bold"
```

Comment:
This change impact several existing bottom sheets, is it OK?

Reply:
It impacts both Join rule settings, and history readability, it should be bolded on both 👍